### PR TITLE
show team names when detecting plagiarism

### LINKED ISSUE #3509: Plagiarism detection: Submittees not shown for Team Exercises
### Describe the bug

When running plagiarism detection on a Team Exercise, tested for Text Exercise, the system does not display who the submission belongs to.

#### To Reproduce

1. Run a plagiarism detection on a Team Exercise
2. Select any match
3. The system displays "unknown" instead of the team

#### Expected behavior

The system shows which team created the submissions that were matched.

#### Screenshots

image

### Environment

<details><pre>
 - OS: Windows 10
 - Browser: Chrome
 - Version 91.0.4472.77 (Offizieller Build) (64-Bit)
</pre></details>

#### Additional context

This issue seems to affect all Team exercises.

#### Output in Web Console

<details><pre>
  No errors / warnings
</pre></details>

### PR DESCRIPTION
**Pull request by joschahenningsen**
### Checklist
- [X] I tested *all* changes and *all* related features with different users (student, tutor, editor, instructor, admin) on the test server https://artemistest.ase.in.tum.de.
- [x] Server: I followed the [coding and design guidelines](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/server.html).
- [x] Server: I documented the Java code using JavaDoc style.

### Motivation and Context

fixes #3509

### Steps for Testing

1. Check plagiarism on text and programming exercises (team and normal).
2. Check if team and student ids are shown.

### CODE REVIEW

**Review by FuchsDominik — APPROVED** [review]
Code-wise approve, changes are good 👍

**Review by nicolasruscher — APPROVED** [review]
Code looks good to me

File: src/main/java/de/tum/in/www1/artemis/service/plagiarism/TextPlagiarismDetectionService.java [hunk 1]
```diff
@@ -64,10 +66,10 @@ public TextPlagiarismDetectionService(TextSubmissionExportService textSubmission
     /**
      * Download all submissions of the exercise, run JPlag, and return the result
      *
-     * @param textExercise to detect plagiarism for
+     * @param textExercise        to detect plagiarism for
      * @param similarityThreshold ignore comparisons whose similarity is below this threshold (%)
-     * @param minimumScore consider only submissions whose score is greater or equal to this value
-     * @param minimumSize consider only submissions whose size is greater or equal to this value
+     * @param minimumScore        consider only submissions whose score is greater or equal to this value
+     * @param minimumSize         consider only submissions whose size is greater or equal to this value
```
Comment (ndricimrr) [inline src/main/java/de/tum/in/www1/artemis/service/plagiarism/TextPlagiarismDetectionService.java:72]: same here regarding spacing. or is it something new, I maybe don't know about 🤔

File: src/main/java/de/tum/in/www1/artemis/service/plagiarism/TextPlagiarismDetectionService.java [hunk 2]
```diff
@@ -64,10 +66,10 @@ public TextPlagiarismDetectionService(TextSubmissionExportService textSubmission
     /**
      * Download all submissions of the exercise, run JPlag, and return the result
      *
-     * @param textExercise to detect plagiarism for
+     * @param textExercise        to detect plagiarism for
```
Comment (ndricimrr) [inline src/main/java/de/tum/in/www1/artemis/service/plagiarism/TextPlagiarismDetectionService.java:69]: spacing.

**Review by ndricimrr — COMMENTED** [review]
Tested on TS1, Works as expexted 👍 

Also some minor code suggestion.

File: src/main/java/de/tum/in/www1/artemis/service/plagiarism/TextPlagiarismDetectionService.java [hunk 3]
```diff
@@ -44,8 +46,8 @@ public TextPlagiarismDetectionService(TextSubmissionExportService textSubmission
      * Reduce a TextExercise Object to a list of latest text submissions. Filters the empty ones because they do not need to be compared
      *
      * @param exerciseWithParticipationsAndSubmissions TextExercise with fetched participations and submissions
-     * @param minimumScore consider only submissions whose score is greater or equal to this value
-     * @param minimumSize consider only submissions whose number of words is greater or equal to this value
+     * @param minimumScore                             consider only submissions whose score is greater or equal to this value
+     * @param minimumSize                              consider only submissions whose number of words is greater or equal to this value
```
[the remark this answers is not in the document]
Comment (joschahenningsen) [reply src/main/java/de/tum/in/www1/artemis/service/plagiarism/TextPlagiarismDetectionService.java:50]: Surely not necessary, IntelliJ did this to improve readability, I'm fine with it.

[the remark this answers is not in the document]
Comment (joschahenningsen) [reply src/main/java/de/tum/in/www1/artemis/service/plagiarism/TextPlagiarismDetectionService.java:50]: But if you want I can revert them

[the remark this answers is not in the document]
Comment (ndricimrr) [reply src/main/java/de/tum/in/www1/artemis/service/plagiarism/TextPlagiarismDetectionService.java:50]: I find them a bit much, but yet I'm not a guideline expert. What do the others think?

[the remark this answers is not in the document]
Comment (joschahenningsen) [reply src/main/java/de/tum/in/www1/artemis/service/plagiarism/TextPlagiarismDetectionService.java:50]: I think they conform with the recommended javadoc styleguide:
https://www.oracle.com/de/technical-resources/articles/java/javadoc-tool.html#format

[the remark this answers is not in the document]
Comment (ndricimrr) [reply src/main/java/de/tum/in/www1/artemis/service/plagiarism/TextPlagiarismDetectionService.java:50]: Oh I see now 👍

**Review by ndricimrr — APPROVED** [review]
Looks good 👍

**Review by canyelten — APPROVED** [review]
Tested on TS1. Student and team names were visible.

**Review by jpbernius — APPROVED** [review]

**Pull request merged**